snorkel/labeling/model/label_model.py

```diff
@@ -304,37 +305,28 @@ def _get_conditional_probs(self, source: Optional[int] = None) -> np.ndarray:
         else:
             return c_probs
 
-    def get_accuracies(self, probs: Optional[np.ndarray] = None) -> np.ndarray:
+    def get_accuracies(self) -> np.ndarray:
         """Return the vector of LF accuracies.
 
-        Parameters
-        ----------
-        probs
-            Conditional probabilities, by default None
-
         Returns
         -------
         np.ndarray
             [m,1] vector of LF accuracies
 
         Example
         -------
-        >>> L = sparse.csr_matrix([[1, 1, 0], [2, 2, 0], [1, 1, 0]])
+        >>> L = sparse.csr_matrix([[1, 1], [2, 2], [1, 1]])
         >>> label_model = LabelModel(verbose=False)
         >>> label_model.train_model(L)
         >>> np.around(label_model.get_accuracies(), 2)
-        array([0.9 , 0.9 , 0.01])
+        array([0.9, 0.9])
         """
         accs = np.zeros(self.m)
         for i in range(self.m):
-            if probs is None:
-                cps = self._get_conditional_probs(source=i)[1:, :]
-            else:
-                cps = probs[
-                    i * (self.cardinality + 1) : (i + 1) * (self.cardinality + 1)
-                ][1:, :]
+            cps = self._get_conditional_probs(source=i)[1:, :]
             accs[i] = np.diag(cps @ self.P.numpy()).sum()
-        return accs
+
+        return np.clip(accs / self.coverage, 0.01, 1.0)
```

Comment:
Wait what's this for?

Reply:
the clip

Reply:
Sometimes the accuracies/coverage can become 1.0001 etc - to prevent that. There’s a similar value clip for cond. probs as well

Reply:
The lower bound is the one I don't understand. If there's some divide-by-zero concern, let's at least do 1e-6